### PR TITLE
Use the same session for the bot and graphql

### LINKED ISSUE #240: Add wolfram Alpha command aliases
## Summary
The only command used for wolfram alpha is `.wolfram`, we should add some easy aliases for this command including `.wa` and `.wolframalpha`

### LINKED ISSUE #249: Update version command to display package release hash
## Summary 
Currently the `.version` command displays a different version/hash than is displayed on the package/release tag. In order to make debugging easier and for there to be clear meaning to the version number, the version should follow the following format `<major_version_number.release_tag/hash`

So, as of writing this, the current friendo version would be displayed as `1.e8ccd8b` 

image

### PR DESCRIPTION
**Pull request by ChrisLovering**
Closes #240 Closes #249 

This also changes the GraphQL cog to use the bot's session, makes the bot work on windows hosts, and makes the import order consistent across the bot.

### CODE REVIEW

**Review by Xithrius — APPROVED** [review]
ya

File: bot/cogs/utilities.py [hunk 1]
```diff
@@ -129,14 +128,9 @@ async def after_create_reminder() -> None:
...
-    async def version(self, ctx: Context) -> str:
+    async def version(self, ctx: Context) -> None:
         """Creates a version number from settings.VERSION and most recent commit hash."""
-        commit_hash = (subprocess.check_output(["git", "rev-parse", "HEAD"]).strip().decode("ascii"))
-        msg = f"Version is {settings.VERSION}{commit_hash[-4:]}"
-
-        await ctx.send(msg)
-
-        return msg
+        await ctx.send(f"Version is {settings.VERSION}+{settings.GIT_SHA}")
```
Comment (fisher60) [inline bot/cogs/utilities.py:133]:
```suggestion
        await ctx.send(f"Version is {settings.VERSION}.{settings.GIT_SHA}")
```

**Review by fisher60 — CHANGES_REQUESTED** [review]
Looks good to me, just need to fix the version format

**Review by fisher60 — APPROVED** [review]
LGTM

**Pull request merged**